chef.py

```diff
@@ -94,7 +47,7 @@ class AfricanStorybookChef(SushiChef):
     """
     channel_info = {
         'CHANNEL_SOURCE_DOMAIN': "www.africanstorybook.org",
-        'CHANNEL_SOURCE_ID': "african-storybook",
+        'CHANNEL_SOURCE_ID': "african-storybook-test",
```

Comment:
Make sure you use the new channel_id when you update the demo server.
channel_id is hash(domain (+) source_id)